toolchain/sem_ir/file.h

```diff
@@ -88,9 +88,6 @@ class File : public Printable<File> {
   // compute this information.
   auto GetIntTypeInfo(TypeId int_type_id) const -> IntTypeInfo {
     auto inst_id = types().GetInstId(int_type_id);
-    if (inst_id == InstId::BuiltinIntType) {
```

Comment:
Ah, I'd removed these TODOs, but that ended up in the next PR branch rather than this one. Now they're removed in this PR instead.

Given the existence of the `IntLiteral` type, which is an integer type that's not an `IntType`, I don't think the TODOs are appropriate. I did add a TODO here to move this function to `TypeStore`, though -- I think it should either go there or become a non-member function given that we have been trying to avoid `File` accumulating a bunch of utility methods.